src/components/FITB_Blank.tsx

```diff
@@ -1,10 +1,12 @@
 import { TextField, styled } from '@mui/material'
-import React, { useState } from 'react'
+import React, { useContext, useState } from 'react'
+
+import { FITBContext, FITBState } from './FITBBlock'
 
 // removes spaces, and checks for equality ignoring caps
 function stringsAreEqual(a: string, b: string): boolean {
-  a = a.replace(' ', '').toLowerCase()
-  b = b.replace(' ', '').toLowerCase()
+  a = a.replace(/ /gi, '').toLowerCase()
+  b = b.replace(/ /gi, '').toLowerCase()
   return a == b
 }
```

Comment:
This might not always be ideal for checking the answer, but we can iterate on that later.